modules/KalturaSupport/components/volumeControl.js

```diff
@@ -158,6 +158,7 @@ mw.PluginManager.add( 'volumeControl', mw.KBaseComponent.extend({
 			var title = gM('mwe-embedplayer-volume-value', percent );
             var $slider = this.getSlider().find('a');
             this.setAccessibility($slider, title);
+            this.updateVolumeUI(percent/100);
```

Comment:
@MaksymKorniiov  
Did we check if the volume comes from the user cookie it is being updated accordingly?

Reply:
@eitanavgil Yes